jpa/keyset-pagination/boot-data-window-pagination/src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -1,5 +1,5 @@
-databaseChangeLog:
-  - includeAll:
-      path: migration/
-      errorIfMissingOrEmpty: true
+databaseChangeLog:
```

yamllint

[error] wrong new line character: expected \n

(new-lines)